examples/pipeline_wavernn/wavernn_inference_wrapper.py

```diff
@@ -166,8 +160,8 @@ def forward(self,
                 loss_name: str = "crossentropy",
                 mulaw: bool = True,
                 batched: bool = True,
-                timesteps: int = 11000,
-                overlap: int = 550) -> Tensor:
+                timesteps: int = 100,
+                overlap: int = 5) -> Tensor:
```

Comment:
nit: update the default values in the doc.

Also could you update the ``shape`` term in the docs? So that we can keep it in consistent with CONTRIBUTING.md.

Reply:
Thanks for the reminder, I've changed all `with shape` to `of size`.

Reply:
Thanks for the reminder, I've changed all `with shape` to `of size`.